packages/@okta/i18n/src/properties/login.properties

```diff
@@ -801,6 +801,16 @@ oie.webauthn.error.not.supported = Security key or biometric authenticator is no
 oie.verify.webauth.title = Verify with Security Key or Biometric Authenticator
 oie.verify.webauthn.instructions = You will be prompted to use a security key or biometric verification (Windows Hello, Touch ID, etc.). Follow the instructions to complete verification.
 oie.verify.webauthn.uv.required.instructions = Biometric verification or a PIN is required to sign in with this authenticator.
+oie.verify.webauthn.cant.verify = Can't verify?
+oie.verify.webauthn.cant.verify.biometric.authenticator.title = Are you trying to use a biometric authenticator?
+oie.verify.webauthn.cant.verify.biometric.authenticator.description1 = Biometric authenticators (fingerprint, face recognition, PIN) will only work on the same device on which they were set up.
+oie.verify.webauthn.cant.verify.biometric.authenticator.description2 = If available, set up another security method that supports verification from a mobile browser.
+oie.verify.webauthn.cant.verify.security.key.title = Are you trying to use a security key?
+oie.verify.webauthn.cant.verify.security.key.description = Insert your security key into a USB port when propmted by the browser and tap on the button or gold disk. Security keys can work on multiple devices.
```

Comment:
This part is redundant with the on in the main description do we need this again ? cc @vincentvoong-okta

Reply:
@magizh-okta which part is redundant? the use of "security key"?

Reply:
@vincentvoong-okta  yeah for security key we just seem to be providing instructions on how to verify and I feel it is not necessary in the 'can't verify' section.  But if we just want to add it for novice users I am fine.

Reply:
@magizh-okta @jasleennuno-okta 
Yeah the security key part was included to let the user know that if they did also enroll in one before they can use it instead of trying to sign in with a platform authenticator. We can change the copy slightly to say: `If you have set up a security key, insert it in a USB port when prompted by the browser and tap on the button or gold disk. Security keys can work on multiple devices.`